src/flash/com/longtailvideo/jwplayer/media/RTMPMediaProvider.as

```diff
@@ -136,8 +136,14 @@
 				}
 				attachNetStream(_stream);
 			}
+
+			media = _video;
+			//If you send buffering events before the provider's media is set
+			//The view will think that the controls should be locked because the provider doesn't have a display.
+			setState(PlayerState.BUFFERING);
```

Comment:
In the previous configuration, these 2 commands would not be executed when we fall through to the final else statement on line 170 : 
```js
    _loader.load(_item.file, XML);
```

Other than that, the logic appears the same. Is this part of the fix?